### ISSUE TITLE
Feature request: netstat net and nstat relevent panels

### ISSUE THREAD

**Issue by realkarmakun** [DamianHurschler/grafana_dashboard_telegraf#2]
Would be nice to have as this dashboard is only one that's working with Flux in newer versions of grafana. I actually have beein migrating them to this specific dashboard so I can provide the queries for you. (I took those from)

Flux powered:
![изображение](https://user-images.githubusercontent.com/20980281/218551556-6ef674d1-87d6-4a42-8039-e1d05d91100c.png)

Old dashboard (the most popular on Grafana dashboard collections):
![изображение](https://user-images.githubusercontent.com/20980281/218551724-a81478ce-61be-4613-954c-90d950d2f599.png)

Query:
```
from(bucket: "${bucket}")
  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
  |> filter(fn: (r) => r["_measurement"]== "netstat")
  |> filter(fn: (r) => r["host"] == "${host}")
  |> filter(fn: (r) => r["_field"]== "tcp_close" or r["_field"]== "tcp_close_wait" or r["_field"] == "tcp_closing" or r["_field"] == "tcp_established" or r["_field"] == "tcp_fin_wait1" or r["_field"] == "tcp_fin_wait2" or r["_field"] == "tcp_last_ack" or r["_field"] == "tcp_syn_recv" or r["_field"] == "tcp_syn_sent" or r["_field"] == "tcp_time_wait")
  |> aggregateWindow(every: v.windowPeriod, fn: mean)
```

Feel free to add it on change it as you want :) (I just started working with flux a day ago so code might me junky)
Also I don't think Grafana Dashboard json is compatible with git so I decided to make issue and not PR

Also have a few more panels, let me now if you are intrested I'll share them as well

**Comment by DamianHurschler** [comment]
Wow, thank you for the support. I'll see how to include them in future releases. And yes, if you have more, please send them through, so I better understand what you're after.
To be honest, I've seen these metrics, but for my application they didn't seem to add anything. Let's see once they are included...

**Comment by DamianHurschler** [comment]
netstat has been included for v3.0. Thx for the proposal with query and picture!
I'll investigate the nstat some more, currently I'm not sure how to include it in a useful form.
